entries/f/freedommobile.ca.json

```diff
@@ -0,0 +1,15 @@
+{
+  "Freedom Mobile": {
```

Comment:
I see that you haven't add any documentation link. Could you maybe find one that describes how to enable 2FA on the site?

If no documentation is available or you can't find it, could you please add some screenshots of the 2FA setup process and/or login process?
__Remember to blur or crop out any personal information linked to you.__

You can attach the screenshots here in a comment, like this:
![screenshot](https://user-images.githubusercontent.com/3535780/62257740-32b4ef00-b407-11e9-8f35-fc7c47ad72fb.gif)